src/datastore/pg-store.ts

```diff
@@ -1467,6 +1471,10 @@ export class PgStore extends BasePgStore {
       const contractIdFilterSql = contractId
         ? sql`AND contract_call_contract_id = ${contractId}`
         : sql``;
+
+      const searchTermFilterSql = searchTerm
+        ? sql`AND similarity(contract_call_function_name, ${searchTerm}) > ${TRGM_SIMILARITY_THRESHOLD}`
```

Comment:
Have you been able to test this `similarity` performance on our staging environments?

Reply:
Yep, I got the same result but with more forgiving match (allowing more typos, etc.)